controller_manager/src/controller_manager_parameters.yaml

```
@@ -25,6 +25,12 @@ controller_manager:
       }
     }
 
+    shutdown_on_initial_state_failure: {
+      type: bool,
+      default_value: true,
```

```suggestion
      default_value: false,
```

We can change it to false for rolling and let's see for jazzy on backport